assets/js/script.js

```diff
@@ -119,209 +102,214 @@ var randomQuestion;
 var currentQuestion;
 var questionCount = 0;
 
-var start = document.getElementById("start-btn");
-var secondsLeft = document.querySelector(".time");
-var container = document.querySelectorAll(".container");
-var scoreBoard = document.querySelector(".score");
-var questionNum = document.querySelector(".question-number");
-var questionEl = document.getElementById("question");
-var nextBtn = document.querySelector(".next-btn");
+var start = document.getElementById('start-btn');
+var secondsLeft = document.querySelector('.time');
+var container = document.querySelectorAll('.container');
+var scoreBoard = document.querySelector('.score');
+var questionNum = document.querySelector('.question-number');
+var questionEl = document.getElementById('question');
+var nextBtn = document.querySelector('.next-btn');
 var answerBtns = document.querySelector('#answer-buttons');
-var choiceA = document.querySelector(".choice-a");
-var choiceB = document.querySelector(".choice-b");
-var choiceC = document.querySelector(".choice-c");
-var choiceD = document.querySelector(".choice-d");
-var selectedAnswer = document.querySelectorAll(".answer-btn");
+var choiceA = document.querySelector('.choice-a');
+var choiceB = document.querySelector('.choice-b');
+var choiceC = document.querySelector('.choice-c');
+var choiceD = document.querySelector('.choice-d');
+var selectedAnswer = document.querySelectorAll('.answer-btn');
 
-var contrastToggle = document.getElementById("contrast");
-var body = document.querySelector("body");
+var correctSound = document.getElementById('correct-sound');
+var wrongSound = document.getElementById('incorrect-sound');
+
+var contrastToggle = document.getElementById('contrast');
+var body = document.querySelector('body');
 
 // for day/night view toggle
-contrastToggle.addEventListener("click", function() {
-    body.classList.toggle("dark");
+contrastToggle.addEventListener('click', function () {
+  body.classList.toggle('dark');
 });
 
 // for background animation
 function ready(document) {
-    document.querySelector(".wrapper").append("<ul class='circles'><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li></ul>");
-};
+  document
+    .querySelector('.wrapper')
+    .append(
+      "<ul class='circles'><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li></ul>"
```

Comment:
consider formatting this on different lines - this is very hard to ready (not sure if JS will allow this format)

```
"<ul class='circles'>
  <li></li>
  <li></li>
  <li></li>
  <li></li>
  <li></li>
  <li></li>...
"
```

Another (better) option is to append the `<li>` in a for loop so you can easily modify them

```
var numLi = 10
for i = 0; i < numLi; i++ {
  ul.append("<li></li>")
}
```

super easy to know and update the count. Similarly if these are related to the number of questions, you have an easy way to making sure they align:

```
var numLi = questions.length // or similar javascript equivalent
```